tests/conversion

```diff
@@ -0,0 +1,264 @@
+# executed on the localhost. For container, lxd-migrate will be installed and executed from
+# an existing container.
+conversion() {
+    instType=$1
+    instName=$2
+    poolType=$3
+    imgPath=$4
+
+    if [ "${instType}" = "vm" ]; then
+        echo "==> TEST: Conversion: Import virtual-machine '${instName}' on '${poolType}' storage pool"
+
+        poolName="conversion-${poolType}"
+        hostAddr="127.0.0.1"
+        instTypeCode="2" # VM in migration questioneer.
+
+        lxdMigrateCmd="lxd-migrate --conversion format"
+
+        # Ensure instance and storage pool with the given name do not exist.
+        lxc delete -f "${instName}" 2>/dev/null || true
+        lxc storage delete "${poolName}" 2>/dev/null || true
+
+        # Create storage pool.
+        lxc storage create "${poolName}" "${poolType}"
+
+    elif [ "${instType}" = "container" ]; then
+        echo "==> TEST: Conversion: Import container '${instName}'"
+
+        poolName=""
+        hostAddr=$(lxc network show lxdbr0 | awk '/ipv4.address:/ {split($2, arr, "/"); print arr[1]}')
+        instTypeCode="1" # Container.
+
+        lxdMigrateCmd="lxc exec ${instName} -- /root/go/bin/lxd-migrate"
+
+        # Install rsync and lxd-migrate.
+        lxc exec "${instName}" -- apt-get update
+        lxc exec "${instName}" -- apt-get install rsync file
+        lxc exec "${instName}" -- snap install go --channel=latest/edge --classic
+        lxc exec "${instName}" -- go install github.com/canonical/lxd/lxd-migrate@latest
+        lxc exec "${instName}" -- /root/go/bin/lxd-migrate --version
+
+        # Set instName to the name of the new container that will be created
+        # from the existing one.
+        instName="${instName}-migration"
+    else
+        echo "Invalid instance type '${instType}'. Valid types are 'container' and 'vm'."
+        return 1
+    fi
+
+    # Generate trust token for conversion.
+    token=$(lxc config trust add --name migrate | awk 'NR==2') || exit 1
+    if [ ! "${token}" ]; then
+        echo "Failed to generate LXD trust token!"
+        return 1
+    fi
+
+    # Migration questions.
+    {
+        if [ "${instType}" = "vm" ]; then
+            echo "n"           # Do not use local unix socket.
+        fi
+
+        echo "${hostAddr}"     # Address of the target LXD server.
+        sleep 1
+        echo "y"               # Yes, this is correct fingerprint.
+        sleep 1
+        echo "1"               # Use a certificate token.
+        echo "${token}"        # Token.
+        echo "${instTypeCode}" # Instace type (1 == container, 2 == virtual-machine).
+
+        if [ "$(lxc project ls -f csv | wc -l)" -gt 1 ]; then
+            echo "default"     # Project name (required if there is more then 1 project)
+        fi
+
+        echo "${instName}"     # Instance name.
+        echo "${imgPath}"      # Local image path (or filesystem path in case of container).
+        echo "no"              # Enable UEFI.
```

Comment:
It's not controlling UEFI vs CSM it seems but more UEFI Secure Boot.

Could we have that disabled only for the Alpine case (assuming that's the only one not supporting SB)?